lib/galaxy/schema/invocation.py

```diff
@@ -591,3 +604,46 @@ class InvocationStepJobsResponseJobModel(InvocationJobsSummaryBaseModel):
 
 class InvocationStepJobsResponseCollectionJobsModel(InvocationJobsSummaryBaseModel):
     model: IMPLICIT_COLLECTION_JOBS_MODEL_CLASS
+
+
+class CreateInvocationFromStore(StoreContentSource):
+    # TODO - add proper description
+    history_id: EncodedDatabaseIdField = Field(default=..., title="History ID", description="")
+    model_config = ConfigDict(extra="allow")
```

Comment:
We should list all fields here and not use `ConfigDict(extra="allow")`

Reply:
So I moved the model to the invocations schema file and it originally had `ConfigDict(extra="allow")`. After tracing where the model is used I noticed that, no fields besides the ones that are already defined are used. For this reason I removed `ConfigDict(extra="allow")`.

Reply:
Thanks for checking, that makes the response model much more usable.